public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -319,7 +319,7 @@ export function requestSupportsSharding(allQueries: LokiQuery[]) {
     .filter((query) => query.queryType !== LokiQueryType.Instant)
     .filter((query) => !query.refId.includes('do-not-shard'))
     .filter((query) => query.expr)
-    .filter((query) => query.direction === LokiQueryDirection.Scan || !isLogsQuery(query.expr));
+    .filter((query) => query.direction === LokiQueryDirection.Scan);
```

Comment:
This is better, but I'm still able to send metric queries with a "scan" direction if I start with scan logs query and then convert to time series query

Reply:
The direction must be sticking afte switching queries. Added an extra test for good measure. Thanks for testing!